torch/_subclasses/schema_check_mode.py

```diff
@@ -42,6 +42,14 @@ def display_ops(self):
         print(*self.ops, sep=",")
 
     def __torch_dispatch__(self, func, types, args=(), kwargs=None):
+        def bitwise_equal(lhs, rhs):
+            if lhs.is_quantized:
```

Comment:
do we actually have this use case? this is tracing a model quantized in eager mode quantization?

Reply:
I was working applying this cross ref test to more operators, and the quantized ones started failing, so I fixed it with this. This is testing code.